### PR TITLE
Styling for guides next/previous navigation

### PR DESCRIPTION
**Pull request by danchamp**
This PR:

- Adds guides SASS partial to the theme
- Styles guide next/previous navigation links

See: [/localgovdrupal/localgov_guides/pull/23](/localgovdrupal/localgov_guides/pull/23)

### CODE REVIEW

File: assets/scss/lib/partials/_guides.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      span {
+        @include rem(left, 7);
+      }
+    }
+
+    &:first-child {
+      margin-right: 1rem;
+    }
+
+    .btn {
```
Comment (cjstevens78) [inline assets/scss/lib/partials/_guides.scss:37]: is this meant to be empty?

Comment (danchamp) [reply assets/scss/lib/partials/_guides.scss]: No, it's a mistake, thanks @cjstevens78. Fixed now.

**Comment by stephen-cox** [comment]
The problem with committing compiled CSS is that we can easily end up with merge conflicts in the CSS. The easiest way to solve this merge conflict is to recompile the CSS on this branch and then merge before anyone else can commit compiled CSS to the master branch.

**Comment by stephen-cox** [comment]
I've fixed the CSS merge conflict as I was doing some theming stuff so merging this.

**Comment by danchamp** [comment]
Thanks @stephen-cox

**Pull request merged**